inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl

```diff
@@ -54,6 +62,9 @@
                 {% if record.authors %}
                   <div class="authors">
                     {{ render_record_authors(is_brief=true, show_affiliations=false) }}
+                    {% if record.get('earliest_date') %}
```

Comment:
Now that you have moved the date to the right side it is not needed there anymore.